### PR TITLE
Add new home page

### PR DESCRIPTION
**Pull request by assertchris**
**needs merged after https://github.com/assertchris/borrow-my-topic/pull/80**

Working Tailwind into the build process, and creating the dynamic sections.

### CODE REVIEW

**Comment by assertchris** [comment]
Ok, folks. I need some serious eyes on this. It's just for the home page, and it's broken the rest of the site's styles. I guess I will need to make extra cards to re-style each page (after I've come up with a nav/form styling I like.

File: app/Http/Controllers/PageController.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class PageController extends Controller
+{
+    public function home()
+    {
+        return view('home');
```
Comment (m1guelpf) [inline app/Http/Controllers/PageController.php]: Route::view to the rescue?

File: app/Http/Controllers/PageController.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    }
+
+    public function landing()
+    {
+        return view('landing');
```
Comment (m1guelpf) [inline app/Http/Controllers/PageController.php]: Route::view to the rescue?

File: routes/web.php [hunk 3]
```diff
@@ -1,21 +1,9 @@
...
-| contains the "web" middleware group. Now create something great!
-|
-*/
-
-Route::view('/', 'landing')->name('landing');
-
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
+Route::get('/', 'PageController@landing')->name('landing');
```
Comment (m1guelpf) [inline routes/web.php]: Route::view to the rescue?

File: routes/web.php [hunk 4]
```diff
@@ -1,21 +1,9 @@
...
+Route::get('/home', 'PageController@home')->name('home')->middleware('auth');
```
Comment (m1guelpf) [inline routes/web.php]: Route::view to the rescue?

Comment (assertchris) [reply routes/web.php]: No, home will be dynamic at some point. It's the user dashboard. We can show all kinds of things there, like open requests.

File: app/Providers/AppServiceProvider.php [hunk 5]
```diff
@@ -12,6 +12,26 @@ public function boot()
         Blade::directive('markdown', function ($expression) {
             return "<?php print Facades\App\Markdown\Converter::render($expression); ?>";
         });
+
+        Blade::directive('link', function ($expression) {
+            $args = eval("return [{$expression}];");
```
Comment (m1guelpf) [inline app/Providers/AppServiceProvider.php]: What does this do?

**Comment by m1guelpf** [comment]
Something looks wrong...
screenshot_2018-03-27-08-07-37

**Comment by assertchris** [comment]
It's fixed now - that was in the middle of a deployment.

**Comment by m1guelpf** [comment]
The only issue I have is that the search bar doesn't look like one. Maybe add a SVG or a placeholder?

**Comment by assertchris** [comment]
good call https://github.com/assertchris/borrow-my-topic/issues/85

**Pull request merged**